db/migrations/20201028220558_board_game.js

```diff
@@ -0,0 +1,16 @@
+exports.up = knex => {
+  return knex.schema.createTable('board_games', function (table) {
+    table.increments('id').primary()
+    table.string('name')
+    table.string('description', 10000)
+    table.string('min_players')
+    table.string('max_players')
+    table.string('image')
+    table.string('average_time').defaultTo(1)
+    table.timestamps(true, true)
```

Comment:
Let me know if you have any ideas for any other relevant information we could add

Reply:
more than an idea for another column, I am still a little bit unsure when to use `table.string` and `table.text`. I think it would be more semantic to use `table.text` when we use content to describe or comment. Let me know what you all think.

What I have found is that we can use `table.text` in this case for line `5`

Reply:
I will try this out and see if the data seeds properly; If it does, i'll push a fix with changing string to text.

Reply:
Yeah, here is what I found in my own research:

`The difference between string and text is the intended size: string is typically of shorter, fixed, length, while text implies a longer, variable-length, string.`

So, I'd say anything that is longer than a sentence  or two should probably be `table.text('column_name')`